crates/field/src/arch/portable/packed.rs

```diff
@@ -456,6 +460,31 @@ macro_rules! impl_ops_for_zero_height {
 
 pub(crate) use impl_ops_for_zero_height;
 
+macro_rules! serialize_deserialize {
```

Comment:
This macro is correct for the canonical tower only, so let's rename to smth like `impl_serialize_deserialize_for_packed_canonical`